cmd/analyzer/subcmds/explain.go

```diff
@@ -53,6 +54,7 @@ func NewExplainCommand(args *inArgs) *cobra.Command {
 	cmd.Flags().Int64Var(&args.eSrcMaxPort, srcMaxPortFlag, netp.MaxPort, "maximum source port for connection description")
 	cmd.Flags().Int64Var(&args.eDstMinPort, dstMinPortFlag, netp.MinPort, "minimum destination port for connection description")
 	cmd.Flags().Int64Var(&args.eDstMaxPort, dstMaxPortFlag, netp.MaxPort, "maximum destination port for connection description")
+	cmd.Flags().BoolVar(&args.detailExplain, detailFlag, false, "adds a section with a list of all relevant allow/deny rules")
```

Comment:
Didn't we agree that the default will be true?
I think it should be true

Reply:
ziv asked for false (after karen comment) we can discuss it again

Reply:
ok, lets discuss